.github/workflows/release.yaml

```diff
@@ -7,43 +7,42 @@ concurrency:
 jobs:
   release:
     if: (github.event_name == 'create' && github.event.ref_type == 'tag') || github.event_name == 'merge_group'
-    name: Release
+    name: Release ${{ matrix.toolchain }} ${{ matrix.platform.target }}
```

Comment:
I think it is better to keep the `${{ matrix.toolchain }}` out of the name of the workflow, so that we don't need to adjust required workflow list every time there is a toolchain change. Anyway, we won't release binaries with multiple versions of toolchain.

Reply:
done